scripts/helm3-init

```diff
@@ -18,7 +18,7 @@ prepare-namespace() {
   kubectl create ns "$KUBE_NAMESPACE" || true
 }
 
-create-ns-and-developer-roles () {
+create-ns-and-developer-roles() {
```

Comment:
I would change the name with `create-ns-and-developer-role-bindings`. What do you think?